ui/src/main/res/values/styles.xml

```diff
@@ -120,9 +120,7 @@
         <item name="cardPreventCornerOverlap">true</item>
     </style>
 
-
-    <style name="Style.Button.Base" parent="Base.Widget.AppCompat.Button">
-        <item name="colorButtonNormal">?attr/colorAccentButton</item>
+    <style name="RatingBar">
+        <item name="colorControlNormal">#ffd740</item>
```

Comment:
They aren't connected. Just delet one code and write another on same place.
I will change color to reference of color